Tao_Current.txt

```diff
@@ -389,9 +446,13 @@ BOF meetings have a very different tone than do WG meetings. The purpose of a BO
 An advantage of having a draft before the BOF is to help focus the discussion. On the other hand, having a draft might tend to limit what the other folks in the BOF want to do in the charter. It's important to remember that most BOFs are held in order to get support for an eventual Working Group, not to get support for a particular document.
 
 Many BOFs don't turn into WGs for a variety of reasons. A common problem is that not enough people can agree on a focus for the work. Another typical reason is that the work wouldn't end up being a standard — if, for example, the document authors don't really want to relinquish change control to a WG. (We'll discuss change control later in this document.) Only two meetings of a BOF can be scheduled on a particular subject; either a WG has to form or the topic should be dropped.
+
 6. RFCs and Internet-Drafts
 
-If you're a new IETF participant and are looking for a particular RFC or Internet-Draft, go to the RFC Editor's web pages, http://www.rfc-editor.org/rfc.html. That site also has links to other RFC collections, many with search capabilities. If you know the number of the RFC you're looking for, go to the RFC Editor's RFC pages, http://www.rfc-editor.org/rfc.html. For Internet-Drafts, a good resource is the IETF web site, https://datatracker.ietf.org/doc, where you can search by title and keyword.
+This section discusses Internet-Drafts and RFCs in the IETF stream, that is, it describes how documents are produced and advanced within the IETF. For a brief note on other RFC streams, see Section 2.2.5.
```

Comment:
At this point, I think we should probably be recommending the tools site or the datatracker for RFCs or I-Ds. They're much more usable.

Reply:
If you're a new IETF participant and are looking for a particular RFC or Internet-Draft, go to the IETF Datatracker [https://datatracker.ietf.org/]. That site has many search capabilities and can help you find the right document and information about its status, dependencies, potential updates and other information. Another entry point for searching and navigating RFCs is Mark Nottingham's EveryRFC [https://everyrfc.org/].